### ISSUE TITLE
No Jupyter Notebook for Chapter 6 / Act 2: LPC-P1114 Readout Protection Bypass

### ISSUE THREAD

**Issue by mringwal** [HardwareHackingHandbook/notebooks#2]
Hi! The book is great.

Would you have or could you provide the Jupyter Notebook for this excercise? The text contains a reference to ti (e.g .page 201 under "Device Setup": "..The following examples references the companion Jupyter Notebook provides..")

Thanks!

**Comment by colinoflynn** [comment]
Oops - let me see what we did with it, we may have forgotten to include this in the final dump (I remember being responsible for this one so by "we" I mean I think it's my fault... oops).

For reference - there is a version of it at https://github.com/newaetech/chipwhisperer-jupyter/blob/7e1df5b6cc8643d3f5de3800468eb93ef3131c0d/courses/faultapp1/LPC1114_Fuse_Bypass.ipynb (this version uses a CW-Lite only, there was a version in the book that could run on a CW-Nano so is slightly different).

**Comment by mringwal** [comment]
Thanks for your quick reply. A first look at the CW-Lite version also reveals where the `CWDevice` is coming from - I got stuck there trying to just type in the code manually. A version for the CW-Nano would be great (I bought that one to get my toes wet...)

**Comment by colinoflynn** [comment]
The CW-Nano is fairly different in the glitch setup unfortunately, so indeed it won't "just work" (unlike some of the SPA/DPA notebooks which do "just work"). But we did have the CW-Nano working so it should be around, I'll update this when I confirm this (hopefully in a couple hours when into office).

**Comment by colinoflynn** [comment]
Here was the file - this was used to create figures in the book so you can see the 'expected output' in this copy. I had a cleaned-up version for upload but this may actually be more helpful in retrospect (since it matches some of the book figures)!

So I'll do a better clean-up, in the meantime hopefully this gets you going.

[Fault_6-LPC1114_Fuse_Bypass with CWNano.zip](https://github.com/HardwareHackingHandbook/notebooks/files/8158622/Fault_6-LPC1114_Fuse_Bypass.with.CWNano.zip)

**Comment by mringwal** [comment]
Thanks a lot for the Notebook incl all output - so I even have access to 'ground truth' for this excercise. O

Ok, I'll start updating the power supply of the dev kit as described in the book and the CW-Lite Notebook (which has color images) next and get going.

I guess the issue should stay open until the cleaned-up version has been added to this repo. Cheers!

**Comment by colinoflynn** [comment]
Yup - please keep issue open, as the notebook I gave you wasn't the "final" version. If you run into issues drop me a note here, I didn't try running it again either (will have to set the HW back up for that).

**Comment by Grazfather** [comment]
Just stumbled upon this. Would be nice to have it committed to the repo!
